### ISSUE TITLE
Rails 6 support

### ISSUE THREAD

**Issue by faraquet** [torba-rb/torba-rails#8]
```
Bundler could not find compatible versions for gem "thor":
  In Gemfile:
    rails (= 6.1.2) was resolved to 6.1.2, which depends on
      railties (= 6.1.2) was resolved to 6.1.2, which depends on
        thor (~> 1.0)

    torba-rails (~> 1.0) was resolved to 1.0.2, which depends on
      torba (~> 1.0) was resolved to 1.0.1, which depends on
        thor (~> 0.19.1)
```

**Comment by nashbridges** [comment]
Please, use just released [Torba v1.1.0](https://github.com/torba-rb/torba/releases/tag/v1.1.0).

**Comment by faraquet** [comment]
Thanks! :+1:
